web/elm/src/Dashboard/Dashboard.elm

```diff
@@ -193,24 +198,67 @@ handleCallback msg ( model, effects ) =
         PipelineToggled _ (Ok ()) ->
             ( model, effects ++ [ FetchData ] )
 
-        PipelineToggled _ (Err err) ->
-            case err of
-                Http.BadStatus { status } ->
-                    ( model
-                    , if status.code == 401 then
-                        effects ++ [ RedirectToLogin ]
+        PipelineHidden pipelineId (Ok ()) ->
```

Comment:
Hrrrrrm. Is splitting this in two really the best route? As I recall, we collapsed a bunch of stuff like this into "toggled" messages precisely to prevent this sort of duplication.

Reply:
e2fb201830ba05ff44c24dc0cd849ac497df3cde

Reply:
LGTM!